borg/archive.py

```diff
@@ -589,7 +572,10 @@ def process_file(self, path, st, cache, ignore_inode=False):
                 status = 'U'  # regular file, unchanged
         else:
             status = 'A'  # regular file, added
-        item = {b'path': safe_path}
+        item = {
+            b'path': safe_path,
+            b'link': st.st_nlink > 1,
```

Comment:
Not exactly, it means that this is item is a hard link (nlink > 1) and that it has the chunks. That way the "items" mapping only holds items which are actually needed to recreate a hard link.